source/changelog.txt

```diff
@@ -78,7 +78,10 @@ v2.3.9
   For more information, see the  `January 21, 2025 Node.js security release \
   <https://nodejs.org/en/blog/release/v20.18.2>`__.
 
-- Stricter validation of user input to address vulnerabilities.
+- Stricter validation of user input to address 
+  `CVE 1691 <https://nvd.nist.gov/vuln/detail/CVE-2025-1691>`_, 
+  `CVE 1692 <https://nvd.nist.gov/vuln/detail/CVE-2025-1692>`_, 
+  and `CVE 1693 <https://nvd.nist.gov/vuln/detail/CVE-2025-1693>`__.
```

Comment:
CVE 1691 and 1692 are named hyperlink references (`_`) while CVE 1693 is an anonymous hyperlink reference (`__`).  I'm not sure how Snooty processes the nodes it's getting from Docutils, but it might be good to standardize them as anonymous references (i.e., with two underscores at the end).

Reply:
Thanks for the catch!